### ISSUE TITLE
Feature Request: Sonarr specify episode

### ISSUE THREAD

**Issue by camjac251** [lokenx/plexrequests-meteor#121]
I had suggested this months ago but you turned it down, hopefully you'll reconsider this time for v0.6.

It would be cool to have a popup appear after someone requests a show asking them what episode they last watched, defaulting to (none or haven't watch any), but allowing them to choose an episode in a season that they are caught up to. Then Sonarr could add the show with everything after the episode they chose marked as monitored and automatically search for it all. That would be a nice feature as even right now, shows aren't triggered for (automatic) searching in the current dev version of plexrequests.

**Comment by lokenx** [comment]
Currently it's new versus all (with automatic searching on Sonarr and SickRage if is selected). Once everything is done for v0.6 I'll look into this. I know Sonarr can be granular on adding but I don't believe SickRage can and the option has to work for both.

Alternatively I can expand issues to allow custom text so they could mention in there maybe as an option.

**Comment by camjac251** [comment]
You could add the feature only for Sonarr even if SickRage doesn't support it. People who use Sonarr shouldn't be left out just because those who use Sickrage don't have the option

**Comment by lokenx** [comment]
That would be a bad experience for SickRage users and there just isn't enough demand. I'll see after 0.6 what options exist, but for now it's new/all.

**Comment by camjac251** [comment]
If you want to see an example of the feature in action try https://www.tvshowtime.com

You type in a show that you want to watch and then it pops up with a modal asking you what episode you last watched. You could see how they implemented theirs if it makes it easier.

**Comment by lokenx** [comment]
Cool, I'll check it out when reviewing how best to do it!Biggest catch at moment though is how to handle SickRage.
		
        
		
		 ---- On Thu, 29 Oct 2015 11:40:06 -0400  notifications@github.com  wrote ----If you want to see an example of the feature in action try https://www.tvshowtime.com
You type in a show that you want to watch and then it pops up with a modal asking you what episode you last watched. You could see how they implemented theirs if it makes it easier.

**Comment by camjac251** [comment]
Hope this helps. I talked to markus on the irc channel about it, here's the conversation
http://hastebin.com/ubitumajew.xml

Also their wiki isn't all up to date so it'll be better to ask them directly the specifics of a function for their api

**Comment by dellipse** [comment]
+1
I think this is a good idea as well.  This allows someone to let you know there is an issue with a specific episode so you can fix it.

**Comment by kamikasky** [comment]
This is actually the only reason I haven't migrated to plexrequests. I'm waiting for more fine tuned episode control. Hope this gets implemented!

**Comment by mneumark** [comment]
Yeah, I was sort of hoping to be able to request at least by season. Sometimes there are some VERY old shows and you only want to commit to getting, say, 3 seasons of the show or something.

**Comment by cwhitlow101** [comment]
Yes, it would be cool to at least incorporate the ability to request individual seasons. However, I understand that if it's not possible to do this in Sickrage then those users would get left out. But it might encourage them to switch to Sonaar ;)
Actually, PlexRequests.NET cross platform version does have this incorporated. Though you both have great, but different, features!! :)
